### PR TITLE
Fixing typos, updating language, fixing broken headers

### PR DESCRIPTION
**Pull request by tashimi**
Signed-off-by: Tasha Drew <tasha.drew@gmail.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/partials/docs/_dev-pkgs-binding.html.md.erb [hunk 1]
```diff
@@ -46,7 +48,11 @@ Once you've defined both ends of the contract you can leverage the bind in any o
 
 ## Starting A Consumer
 
+<<<<<<< HEAD
```
Comment (reset) [inline www/source/partials/docs/_dev-pkgs-binding.html.md.erb]: You've got a merge conflict here

File: www/source/partials/docs/_dev-pkgs-plan-builds.html.md.erb [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
 # <a name="plan-builds" id="plan-builds" data-magellan-target="plan-builds">Plan Builds</a>
 
-As part of building a package, it needs to be signed with a secret origin key at buildtime. On your host machine, if you want to generate an origin key pair manually, or you used the `hab setup` and simply need another origin key pair, run the following command:
+Packages need to be signed with a secret origin key at buildtime. Generate an origin key pair manually by running the following command on your host machine:
 
     hab origin key generate originname
 
 The `hab-origin` subcommand will place originname-_timestamp_.sig.key and originname-_timestamp_.pub files (the origin key pair) in the `$HOME/.hab/cache/keys` directory. If you're creating origin keys in the Studio container or you are running as root on a Linux machine, your keys will be stored in `/hab/cache/keys`.
 
+<<<<<<< HEAD
```
Comment (reset) [inline www/source/partials/docs/_dev-pkgs-plan-builds.html.md.erb]: Merge conflict here as well

File: www/source/partials/docs/_dev-pkgs-share.html.md.erb [hunk 3]
```diff
@@ -63,9 +63,15 @@ You can instruct the Supervisor to download and run packages from a depot by usi
 $ hab start core/postgresql
 ```
 
+<<<<<<< HEAD
```
Comment (reset) [inline www/source/partials/docs/_dev-pkgs-share.html.md.erb]: Merge conflict here as well

File: www/source/partials/docs/_bp-iterative-development.html.md.erb [hunk 4]
```diff
@@ -1,16 +1,24 @@
+<<<<<<< HEAD
```
Comment (reset) [inline www/source/partials/docs/_bp-iterative-development.html.md.erb]: Merge conflict here as well and right below

Comment (tashimi) [reply www/source/partials/docs/_bp-iterative-development.html.md.erb]: Ugh I thought I fixed these. Will try again.

**Comment by christophermaier** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/281681580).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/281681580).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**